services/process/standard/parameters.go

```diff
@@ -38,6 +40,7 @@ type parameters struct {
 	peers                peers.Service
 	stores               []e2wtypes.Store
 	generationPassphrase []byte
+	generationTimeout    time.Duration
```

Comment:
Given this is a new parameter, it makes sense to provide a default value for it.  This should be in the section that is currently:

```
   parameters := parameters{
        logLevel:  zerolog.GlobalLevel(),
        encryptor: keystorev4.New(),
    }
```

Note that the old (hard-coded) timeout parameter was 70s, and the new default appears to be 10s.  Is there a reason for this change?

Reply:
I can't get it. Should I provide default value for the `generationTimeout ` param in the `fetchConfig` section or here?

Reply:
The thing to understand here is that the service could be called from outside of the dirk process itself (if imported by another codebase), so you need to have both:

- you have a default in `process/standard/parameters.go` so that if someone is already calling this codebase without the parameter specified it won't break with this change
- you have a default in `fetchConfig()` because you are explicitly setting this value as part of your call to start the process service, and otherwise it will default to 0 and fail

Reply:
Make sense, thanks.